src/segments/azd.go

```diff
@@ -0,0 +1,77 @@
+package segments
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"strings"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type Azd struct {
+	props properties.Properties
+	env   platform.Environment
+
+	AzdConfig
+}
+
+type AzdConfig struct {
+	Version            int    `json:"version"`
+	DefaultEnvironment string `json:"defaultEnvironment"`
+}
+
+func (t *Azd) Template() string {
+	return " {{ .DefaultEnvironment }} "
+}
+
+func (t *Azd) Init(props properties.Properties, env platform.Environment) {
+	t.props = props
+	t.env = env
+}
+
+func (t *Azd) Enabled() bool {
+	var location string
+
+	folders := []string{".azure"}
+	for _, folder := range folders {
+		if file, err := t.env.HasParentFilePath(folder); err == nil {
+			location = file.ParentFolder
+			break
+		}
+	}
+
+	if len(location) == 0 {
+		t.env.Debug("No .azure folder found in parent directories")
+		return false
+	}
+
+	files := t.env.LsDir(filepath.Join(location, ".azure"))
```

Comment:
```suggestion
    dotAzureFolder := filepath.Join(location, ".azure")
	files := t.env.LsDir(dotAzureFolder)
```

`filepath.Join` is repeated